tsl/test/sql/dist_partial_agg.sql.in

```diff
@@ -28,6 +28,17 @@ $$
        CREATE TYPE custom_type AS (high int, low int);
 $$);
 GRANT CREATE ON SCHEMA public TO :ROLE_1;
+
+-- make sure parallel query plans are preferred on data nodes
+ALTER DATABASE :DATA_NODE_1 SET parallel_setup_cost TO 1;
+ALTER DATABASE :DATA_NODE_2 SET parallel_setup_cost TO 1;
+ALTER DATABASE :DATA_NODE_3 SET parallel_setup_cost TO 1;
+
+-- make sure partitionwise aggregation is enabled on data nodes
+ALTER DATABASE :DATA_NODE_1 SET enable_partitionwise_aggregate TO true;
+ALTER DATABASE :DATA_NODE_2 SET enable_partitionwise_aggregate TO true;
+ALTER DATABASE :DATA_NODE_3 SET enable_partitionwise_aggregate TO true;
+
 SET ROLE :ROLE_1;
 SELECT table_name FROM create_distributed_hypertable( 'conditions', 'timec', 'location', 3, chunk_time_interval => INTERVAL '1 day');
```

Comment:
@erimatnor Don't we expect any plan output changes with this?

Reply:
If we would do explains in these tests, with remote_explains=true, we would see that parallel plans are produced on data nodes instead of non-parallel plans.